modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/SW360AuthenticationClient.java

```diff
@@ -26,14 +28,13 @@
 
 public class SW360AuthenticationClient {
     private final String GRANT_TYPE_VALUE = "password";
-    private final String LIFERAY_CLIENT_CREDENTIALS = "trusted-sw360-client:sw360-secret";
+//    private final String LIFERAY_CLIENT_CREDENTIALS = "trusted-sw360-client:sw360-secret";
```

Comment:
commented code

Reply:
Ups, you are right, fixed with the latest update, result of beeing disturbed during work